pywinauto/hooks.py

```diff
@@ -0,0 +1,231 @@
+from ctypes import wintypes
+from ctypes import windll, CFUNCTYPE, POINTER, c_int, c_void_p, byref
+import atexit
+
+
+def create_pointer(handler):
+    cmp_func = CFUNCTYPE(c_int, c_int, c_int, POINTER(c_void_p))
+    return cmp_func(handler)
+
+
+class KeyboardEvent:
+    def __init__(self, current_key=None, event_type=None, pressed_key=[]):
+        self.current_key = current_key
+        self.event_type = event_type
+        self.pressed_key = pressed_key
+
+
+class MouseEvent:
+    def __init__(self, current_key=None, event_type=None):
+        self.current_key = current_key
+        self.event_type = event_type
+
+
+class Hook:
+    MouseId2Key = {513: 'LButton',
+                   514: 'LButton',
+                   516: 'RButton',
+                   517: 'RButton',
+                   519: 'WheelButton',
+                   520: 'WheelButton',
+                   522: 'Wheel'}
+
+    MouseId2EventType = {513: 'key down',
+                         514: 'key up',
+                         516: 'key down',
+                         517: 'key up',
+                         519: 'key down',
+                         520: 'key up',
+                         522: None}
+
+    ID2Key = {8: 'Back',
+              9: 'Tab',
```

Comment:
When I added `from . import SendKeysCtypes as SendKeys` and launched it, i get exception: `SystemError: Parent module '' not loaded, cannot perform relative import`
Do you have any ideas why it happened?